waku/node/v2/wakunode2.nim

```diff
@@ -97,6 +97,8 @@ proc init*(T: type WakuNode, nodeKey: crypto.PrivateKey,
     filters: initTable[string, Filter]()
   )
 
+  # TODO This is _not_ safe. Subscribe should happen in start and after things settled down.
```

Comment:
hmm, this doesn't seem to be doing anything? It just subscribes a handler that does nothing to WakuRelay? Can be removed I guess.
And then similar code but with an actual handler and on "waku" topic is subscribed in start, which will actually do the work it seems.

Reply:
Right it doesn't show up in the diff, the next section is:

```
    # XXX: Is using discard here fine? Not sure if we want init to be async?
    # Can also move this to the start proc, possibly wiser?
    discard result.subscribe(topic, handler)

```